README.md

```diff
@@ -61,3 +61,18 @@ Then run above command again.
 ```sh
 node ../node_modules/db-migrate/bin/db-migrate create name-for-the-migration --sql-file
 ```
+
+## User Sign-Up Password Emailing
+
+To configure the SMTP server, add the following variables to your .env file and set their values based on your provider:
+
+MAIL_HOST=<your_smtp_host>
+MAIL_PORT=<your_smtp_port>
+MAIL_USER=<your_smtp_user>
+MAIL_PASSWORD=<your_smtp_password>
+
+Replace <your_smtp_host>, <your_smtp_port>, <your_smtp_user>, and <your_smtp_password> with the appropriate values provided by your SMTP service.
+
+To run the database, add the following variables to your .env file:
+DB_PASSWORD=opensesame
+DB_USERNAME=postgres
```

Comment:
Create a new section for this, since it's not part of the email configuration

```suggestion
## Local database configuration

To configure your system to connect to the local database, add the following variables to your .env file:

\```
DB_PASSWORD=opensesame
DB_USERNAME=postgres
\```
```